docs/resources/Guild.md

```diff
@@ -9,49 +9,48 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 > info
 > Fields specific to the `GUILD_CREATE` event are listed in the [Gateway documentation](#DOCS_TOPICS_GATEWAY/guild-create).
 
-| Field                         | Type                                                                                                         | Description                                                                                                                                                            |
-| ----------------------------- | ------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I'm not able to comment about the rest of this PR, but we shouldn't remove this header.

Reply:
Are you certain.

Reply:
yes